src/util.js

```diff
@@ -373,6 +373,47 @@ define([
         return div.html();
     };
 
+    that.questionAutocomplete = function (input, form, options) {
+        options = _.defaults(options || {}, {
+            category: 'Question Reference',
+            insertTpl: '${name}',
+            property: '',
+        });
+
+        input.atwho({
+            at: "/data/",
+            data: _.chain(form.getMugList())
+                   .map(function(mug) {
+                        return {
+                            id: mug.ufid,
+                            name: mug.absolutePath,
+                        };
+                    })
+                    .filter(function(choice) { return choice.name; })
+                    .value(),
+            displayTpl: '<li>${name}</li>',
+            insertTpl: options.insertTpl,
+            limit: 10,
+            maxLen: 30,
+            callbacks: {
+                matcher: function(flag, subtext) {
+                    var match, regexp;
+                    regexp = new RegExp('(\\s+|^)' + flag + '([\\w_/]*)$', 'gi');
```

Comment:
This isn't really necessary right now as it's always `/data/` but went ahead and escaped it because I assume it will get generalized at some point and will probably cause me a headache of a bug.